e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.js

```diff
@@ -0,0 +1,136 @@
+import {
+  addCustomColumn,
+  restore,
+  popover,
+  openOrdersTable,
+  expressionEditorWidget,
+} from "e2e/support/helpers";
+
+describe("scenarios > question > custom column > expression shortcuts", () => {
+  beforeEach(() => {
+    cy.intercept("POST", "/api/dataset").as("dataset");
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("should be possible to select a combine columns shortcut", () => {
+    openOrdersTable({ mode: "notebook", limit: 5 });
+    addCustomColumn();
+    selectCombineColumns();
+
+    selectColumn("Total");
+
+    expressionEditorWidget().findByText("Total").should("exist");
+
+    selectColumn("Product", "Rating");
+
+    expressionEditorWidget().within(() => {
+      cy.findByText("Product → Rating").should("exist");
+
+      cy.findByTestId("combine-example").should(
+        "contain",
+        "123.45678901234567 123.45678901234567",
+      );
+
+      cy.findByText(/Separated by/).click();
+      cy.findByLabelText("Separator").type("__");
+
+      cy.findByTestId("combine-example").should(
+        "contain",
+        "123.45678901234567__123.45678901234567",
+      );
+
+      cy.findByText("Done").click();
+
+      cy.findByTestId("expression-editor-textfield").should(
+        "contain",
+        'concat([Total], "__", [Product → Rating])',
+      );
+      cy.findByTestId("expression-name").should(
+        "have.value",
+        "Combined Total, Rating",
+      );
+    });
+  });
+
+  it("should be possible to cancel when using the combine column shortcut", () => {
+    openOrdersTable({ mode: "notebook" });
+    addCustomColumn();
+    selectCombineColumns();
+
+    expressionEditorWidget().within(() => {
+      cy.findByText("Select columns to combine").click();
+    });
+
+    cy.get(".ace_text-input").should("have.value", "\n\n");
+    cy.findByTestId("expression-name").should("have.value", "");
+  });
```

Comment:
This test was super confusing to me, to the extent that I had to run it locally and then observe what's going on.
A part of it is that "Select columns to combine" really doesn't translate to "go back" ([Slack thread](https://metaboat.slack.com/archives/C0645JP1W81/p1714123288649479) about this), but another maybe even more important thing is that we didn't even select a column to begin with. So going back from "no action" to asserting that the custom expression formula is empty provides no confidence, imo.

Can you please at least add one column, but ideally add both and then go back.
The comment above `cy.findByText("Select columns to combine").click();` would help explain what are we doing, but it might be simpler to click on icon `<`.